packages/storage/amplify_storage_s3/example/integration_test/use_case_test.dart

```diff
@@ -56,17 +56,16 @@ void main() {
   testLargeFileBytes[0] = 101;
   testLargeFileBytes[5 * 1024] = 102;
   final testObjectKey1 = 'user1-guest-object-${uuid()}';
-  final testObject1MoveKey = 'user1-guest-object-move-${uuid()}';
   final testObjectKey2 = 'user1-protected-object-${uuid()}';
-  final testObject2MoveKey = 'user1-protected-object-move-${uuid()}';
   final testObjectKey3 = 'user1-private-object-${uuid()}';
   final testObject3CopyKey = 'user1-private-object-copy-${uuid()}';
   final testObject3CopyMoveKey = 'user1-private-object-copy-move-${uuid()}';
   const testObjectFileName1 = 'user1Guest.txt';
   const testObjectFileName2 = 'user1Protected.jpg';
   const testObjectFileName3 = 'user1Private.large';
 
-  for (final entry in amplifyEnvironments.entries) {
+  for (final entry in amplifyEnvironments.entries
+      .where((element) => element.key != 'custom-prefix')) {
```

Comment:
are we going to eventually add back in the custom-prefix environment?

Reply:
I don't see any benefit to adding it back. With the changes in this PR an environment that has paths `public/`, `protected/`, and `private/` is not different that an env that has path `foo/`, `bar/`, and `baz/`.

There are an infinite number of backends that could be created. We cannot test against all of them. The standard config ("main") already covers an env that has paths with and without identity ids.